manifests/init.pp

```diff
@@ -80,6 +80,9 @@
   $vas_conf_libvas_site_only_servers                    = false,
   $vas_conf_libvas_use_dns_srv                          = true,
   $vas_conf_libvas_use_tcp_only                         = true,
+  $vas_conf_libvas_crypt_thingys                        = [ 'default_tgs_enctypes', 'aes256-cts-hmac-sha1-96',
+                                                            'default_tkt_enctypes', 'aes256-cts-hmac-sha1-96',
+                                                            'default_etypes',       'aes256-cts-hmac-sha1-96' ],
```

Comment:
Please use a hash instead to make the data more structured or use independent params for each type.